src/components/proofs/EmailProof.tsx

```diff
@@ -144,6 +162,7 @@ export default function () {
         {open && (
           <EmailProofForm
             domain={domain}
+            token={EmailDomainStore.emailDomain === defaultDomain ? token : ''}
```

Comment:
Why don't we use `emailDomain` here? Without accessing `EmailDomainStore` I mean

Reply:
Changed it, thx